great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -523,31 +559,37 @@ def add_column_row_condition(
         self, domain_kwargs, column_name=None, filter_null=True, filter_nan=False
     ):
         # We explicitly handle filter_nan & filter_null for spark using a spark-native condition
-        if "row_condition" in domain_kwargs and domain_kwargs["row_condition"]:
```

Comment:
@anthonyburdi Where does `row_condition` go now?  Thanks.

Reply:
It stays in `get_domain_records`, and moved above `filter_conditions` handling as discussed.